Source/Diagnostics/FlushFormats/FlushFormatAscent.cpp

```diff
@@ -19,29 +19,48 @@ FlushFormatAscent::WriteToFile (
     bool /*isBTD*/, int /*snapshotID*/, const amrex::Geometry& /*full_BTD_snapshot*/, bool /*isLastBTDFlush*/) const
 {
 #ifdef AMREX_USE_ASCENT
+    WARPX_PROFILE("FlushFormatAscent::WriteToFile()");
+    WARPX_PROFILE_VAR_NS("FlushFormatAscent::WriteToFile::MultiLevelToBlueprint",
+                         prof_ascent_mesh_blueprint);
+    WARPX_PROFILE_VAR_NS("FlushFormatAscent::WriteToFile::WriteParticles",
+                         prof_ascent_particles);
+    WARPX_PROFILE_VAR_NS("FlushFormatAscent::WriteToFile::publish",
+                         prof_ascent_publish);
+    WARPX_PROFILE_VAR_NS("FlushFormatAscent::WriteToFile::execute",
+                         prof_ascent_execute);
+
     auto & warpx = WarpX::GetInstance();
 
     // wrap mesh data
+    WARPX_PROFILE_VAR_START(prof_ascent_mesh_blueprint);
```

Comment:
If we remove the above, we write here:
```suggestion
    WARPX_PROFILE_VAR("FlushFormatAscent::WriteToFile::MultiLevelToBlueprint", prof_ascent_mesh_blueprint);
```